tensilelite/Tensile/KernelWriter.py

```diff
@@ -2288,7 +2289,9 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
     if kernel["PrefetchGlobalRead"]:
       if not kernel["SuppressNoLoadLoop"]:
         gsuLabel = Label(label=self.labels.getNameInc("GSU"), comment="")
-        module.add(SCmpEQU32(src0=sgpr("GSU"), src1=1, comment="GSU == 1 ?"))
+        with self.allocTmpSgpr(1) as tmpSgprGSU:
```

Comment:
same here

Reply:
This can't be done with SAndB32 because SCC=(D!=0) is set in SAndB32. It only compare zero and non-zero cases instead of "==1".